typescript/sdk/src/core/EvmCoreReader.ts

```diff
@@ -54,7 +61,17 @@ export class EvmCoreReader implements CoreReader {
           defaultHook: this.evmHookReader.deriveHookConfig(defaultHook),
           requiredHook: this.evmHookReader.deriveHookConfig(requiredHook),
         },
-        async (_, readerCall) => readerCall,
+        async (_, readerCall) => {
+          try {
+            return await readerCall;
+          } catch (e) {
+            this.logger.error(
+              `EvmCoreReader: readerCall failed for ${address}:`,
+              e,
+            );
+            return;
```

Comment:
What happens when `undefined` is returned here?

Reply:
If there are failures during execution, we continue with the other reader calls

example output:
```
Hyperlane CLI
{"level":30,"time":1719496512888,"pid":1537,"msg":"Your CLI version: 3.15.0, latest version: 3.16.0"}
Hyperlane Core Read
-------------------
Unknown domain ID 421613, skipping domain configuration
Unknown domain ID 5, skipping domain configuration
Unknown domain ID 420, skipping domain configuration
Unknown domain ID 1442, skipping domain configuration
Unknown domain ID 80001, skipping domain configuration
Failed to derive undefined hook (0xC9D50584F08Bf6cCD1004d14c7062044b45E3b48): Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data="0x", transaction={"to":"0xC9D50584F08Bf6cCD1004d14c7062044b45E3b48","data":"0xe445e7dd","accessList":null}, error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":91,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n","error":{"code":-32000},"requestBody":"{\"method\":\"eth_call\",\"params\":[{\"to\":\"0xc9d50584f08bf6ccd1004d14c7062044b45e3b48\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":91,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://alfajores-forno.celo-testnet.org"}, code=CALL_EXCEPTION, version=providers/5.7.2)
    at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
    at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
    at checkError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:108:16)
    at HyperlaneJsonRpcProvider.<anonymous> (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:751:47)
    at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:48:23)
    at Object.throw (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:29:53)
    at rejected (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:21:65)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  reason: 'missing revert data in call exception; Transaction reverted without a reason string',
  code: 'CALL_EXCEPTION',
  data: '0x',
  transaction: {
    to: '0xC9D50584F08Bf6cCD1004d14c7062044b45E3b48',
    data: '0xe445e7dd',
    accessList: null
  },
  error: Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":91,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_call\",\"params\":[{\"to\":\"0xc9d50584f08bf6ccd1004d14c7062044b45e3b48\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":91,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://alfajores-forno.celo-testnet.org", code=SERVER_ERROR, version=web/5.7.0)
      at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
      at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
      at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:308:32
      at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
      at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
      at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
    reason: 'processing response error',
    code: 'SERVER_ERROR',
    body: '{"jsonrpc":"2.0","id":91,"error":{"code":-32000,"message":"execution reverted"}}\n',
    error: Error: execution reverted
        at getResult (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:191:21)
        at processJsonFunc (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:351:22)
        at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:283:46
        at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
        at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
        at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
        at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
      code: -32000,
      data: undefined
    },
    requestBody: '{"method":"eth_call","params":[{"to":"0xc9d50584f08bf6ccd1004d14c7062044b45e3b48","data":"0xe445e7dd"},"latest"],"id":91,"jsonrpc":"2.0"}',
    requestMethod: 'POST',
    url: 'https://alfajores-forno.celo-testnet.org'
  }
}
EvmCoreReader: readerCall failed for 0xEf9F292fcEBC3848bF4bB92a96a04F9ECBb78E59: Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data="0x", transaction={"to":"0xC9D50584F08Bf6cCD1004d14c7062044b45E3b48","data":"0xe445e7dd","accessList":null}, error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":91,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n","error":{"code":-32000},"requestBody":"{\"method\":\"eth_call\",\"params\":[{\"to\":\"0xc9d50584f08bf6ccd1004d14c7062044b45e3b48\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":91,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://alfajores-forno.celo-testnet.org"}, code=CALL_EXCEPTION, version=providers/5.7.2)
    at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
    at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
    at checkError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:108:16)
    at HyperlaneJsonRpcProvider.<anonymous> (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:751:47)
    at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:48:23)
    at Object.throw (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:29:53)
    at rejected (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:21:65)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  reason: 'missing revert data in call exception; Transaction reverted without a reason string',
  code: 'CALL_EXCEPTION',
  data: '0x',
  transaction: {
    to: '0xC9D50584F08Bf6cCD1004d14c7062044b45E3b48',
    data: '0xe445e7dd',
    accessList: null
  },
  error: Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":91,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_call\",\"params\":[{\"to\":\"0xc9d50584f08bf6ccd1004d14c7062044b45e3b48\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":91,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://alfajores-forno.celo-testnet.org", code=SERVER_ERROR, version=web/5.7.0)
      at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
      at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
      at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:308:32
      at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
      at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
      at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
    reason: 'processing response error',
    code: 'SERVER_ERROR',
    body: '{"jsonrpc":"2.0","id":91,"error":{"code":-32000,"message":"execution reverted"}}\n',
    error: Error: execution reverted
        at getResult (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:191:21)
        at processJsonFunc (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:351:22)
        at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:283:46
        at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
        at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
        at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
        at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
      code: -32000,
      data: undefined
    },
    requestBody: '{"method":"eth_call","params":[{"to":"0xc9d50584f08bf6ccd1004d14c7062044b45e3b48","data":"0xe445e7dd"},"latest"],"id":91,"jsonrpc":"2.0"}',
    requestMethod: 'POST',
    url: 'https://alfajores-forno.celo-testnet.org'
  }
}
All providers failed for method call and params {
  "transaction": {
    "to": "0x6e529469D02308250aC253f07e79F5886C05F3c0",
    "data": "0xd5438eae",
    "accessList": null
  },
  "blockTag": "latest"
}
Failed to derive undefined hook (0xeee0A8f87D358672708598714C84f3E2B741019c): Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data="0x", transaction={"to":"0xeee0A8f87D358672708598714C84f3E2B741019c","data":"0xe445e7dd","accessList":null}, error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":121,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n","error":{"code":-32000},"requestBody":"{\"method\":\"eth_call\",\"params\":[{\"to\":\"0xeee0a8f87d358672708598714c84f3e2b741019c\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":121,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://alfajores-forno.celo-testnet.org"}, code=CALL_EXCEPTION, version=providers/5.7.2)
    at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
    at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
    at checkError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:108:16)
    at HyperlaneJsonRpcProvider.<anonymous> (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:751:47)
    at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:48:23)
    at Object.throw (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:29:53)
    at rejected (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:21:65)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  reason: 'missing revert data in call exception; Transaction reverted without a reason string',
  code: 'CALL_EXCEPTION',
  data: '0x',
  transaction: {
    to: '0xeee0A8f87D358672708598714C84f3E2B741019c',
    data: '0xe445e7dd',
    accessList: null
  },
  error: Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":121,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_call\",\"params\":[{\"to\":\"0xeee0a8f87d358672708598714c84f3e2b741019c\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":121,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://alfajores-forno.celo-testnet.org", code=SERVER_ERROR, version=web/5.7.0)
      at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
      at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
      at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:308:32
      at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
      at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
      at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
    reason: 'processing response error',
    code: 'SERVER_ERROR',
    body: '{"jsonrpc":"2.0","id":121,"error":{"code":-32000,"message":"execution reverted"}}\n',
    error: Error: execution reverted
        at getResult (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:191:21)
        at processJsonFunc (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:351:22)
        at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:283:46
        at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
        at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
        at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
        at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
      code: -32000,
      data: undefined
    },
    requestBody: '{"method":"eth_call","params":[{"to":"0xeee0a8f87d358672708598714c84f3e2b741019c","data":"0xe445e7dd"},"latest"],"id":121,"jsonrpc":"2.0"}',
    requestMethod: 'POST',
    url: 'https://alfajores-forno.celo-testnet.org'
  }
}
Failed to derive undefined hook (0xeee0A8f87D358672708598714C84f3E2B741019c): Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data="0x", transaction={"to":"0xeee0A8f87D358672708598714C84f3E2B741019c","data":"0xe445e7dd","accessList":null}, error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":119,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n","error":{"code":-32000},"requestBody":"{\"method\":\"eth_call\",\"params\":[{\"to\":\"0xeee0a8f87d358672708598714c84f3e2b741019c\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":119,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://alfajores-forno.celo-testnet.org"}, code=CALL_EXCEPTION, version=providers/5.7.2)
    at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
    at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
    at checkError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:108:16)
    at HyperlaneJsonRpcProvider.<anonymous> (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:751:47)
    at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:48:23)
    at Object.throw (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:29:53)
    at rejected (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:21:65)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  reason: 'missing revert data in call exception; Transaction reverted without a reason string',
  code: 'CALL_EXCEPTION',
  data: '0x',
  transaction: {
    to: '0xeee0A8f87D358672708598714C84f3E2B741019c',
    data: '0xe445e7dd',
    accessList: null
  },
  error: Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":119,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_call\",\"params\":[{\"to\":\"0xeee0a8f87d358672708598714c84f3e2b741019c\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":119,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://alfajores-forno.celo-testnet.org", code=SERVER_ERROR, version=web/5.7.0)
      at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
      at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
      at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:308:32
      at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
      at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
      at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
    reason: 'processing response error',
    code: 'SERVER_ERROR',
    body: '{"jsonrpc":"2.0","id":119,"error":{"code":-32000,"message":"execution reverted"}}\n',
    error: Error: execution reverted
        at getResult (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:191:21)
        at processJsonFunc (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:351:22)
        at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:283:46
        at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
        at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
        at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
        at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
      code: -32000,
      data: undefined
    },
    requestBody: '{"method":"eth_call","params":[{"to":"0xeee0a8f87d358672708598714c84f3e2b741019c","data":"0xe445e7dd"},"latest"],"id":119,"jsonrpc":"2.0"}',
    requestMethod: 'POST',
    url: 'https://alfajores-forno.celo-testnet.org'
  }
}
Failed to derive undefined hook (0xeee0A8f87D358672708598714C84f3E2B741019c): Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data="0x", transaction={"to":"0xeee0A8f87D358672708598714C84f3E2B741019c","data":"0xe445e7dd","accessList":null}, error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":160,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n","error":{"code":-32000},"requestBody":"{\"method\":\"eth_call\",\"params\":[{\"to\":\"0xeee0a8f87d358672708598714c84f3e2b741019c\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":160,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://alfajores-forno.celo-testnet.org"}, code=CALL_EXCEPTION, version=providers/5.7.2)
    at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
    at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
    at checkError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:108:16)
    at HyperlaneJsonRpcProvider.<anonymous> (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:751:47)
    at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:48:23)
    at Object.throw (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:29:53)
    at rejected (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:21:65)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  reason: 'missing revert data in call exception; Transaction reverted without a reason string',
  code: 'CALL_EXCEPTION',
  data: '0x',
  transaction: {
    to: '0xeee0A8f87D358672708598714C84f3E2B741019c',
    data: '0xe445e7dd',
    accessList: null
  },
  error: Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":160,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_call\",\"params\":[{\"to\":\"0xeee0a8f87d358672708598714c84f3e2b741019c\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":160,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://alfajores-forno.celo-testnet.org", code=SERVER_ERROR, version=web/5.7.0)
      at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
      at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
      at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:308:32
      at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
      at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
      at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
    reason: 'processing response error',
    code: 'SERVER_ERROR',
    body: '{"jsonrpc":"2.0","id":160,"error":{"code":-32000,"message":"execution reverted"}}\n',
    error: Error: execution reverted
        at getResult (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:191:21)
        at processJsonFunc (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:351:22)
        at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:283:46
        at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
        at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
        at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
        at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
      code: -32000,
      data: undefined
    },
    requestBody: '{"method":"eth_call","params":[{"to":"0xeee0a8f87d358672708598714c84f3e2b741019c","data":"0xe445e7dd"},"latest"],"id":160,"jsonrpc":"2.0"}',
    requestMethod: 'POST',
    url: 'https://alfajores-forno.celo-testnet.org'
  }
}
Failed to derive undefined hook (0xeee0A8f87D358672708598714C84f3E2B741019c): Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data="0x", transaction={"to":"0xeee0A8f87D358672708598714C84f3E2B741019c","data":"0xe445e7dd","accessList":null}, error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":159,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n","error":{"code":-32000},"requestBody":"{\"method\":\"eth_call\",\"params\":[{\"to\":\"0xeee0a8f87d358672708598714c84f3e2b741019c\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":159,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://alfajores-forno.celo-testnet.org"}, code=CALL_EXCEPTION, version=providers/5.7.2)
    at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
    at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
    at checkError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:108:16)
    at HyperlaneJsonRpcProvider.<anonymous> (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:751:47)
    at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:48:23)
    at Object.throw (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:29:53)
    at rejected (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:21:65)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  reason: 'missing revert data in call exception; Transaction reverted without a reason string',
  code: 'CALL_EXCEPTION',
  data: '0x',
  transaction: {
    to: '0xeee0A8f87D358672708598714C84f3E2B741019c',
    data: '0xe445e7dd',
    accessList: null
  },
  error: Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":159,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_call\",\"params\":[{\"to\":\"0xeee0a8f87d358672708598714c84f3e2b741019c\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":159,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://alfajores-forno.celo-testnet.org", code=SERVER_ERROR, version=web/5.7.0)
      at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
      at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
      at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:308:32
      at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
      at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
      at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
    reason: 'processing response error',
    code: 'SERVER_ERROR',
    body: '{"jsonrpc":"2.0","id":159,"error":{"code":-32000,"message":"execution reverted"}}\n',
    error: Error: execution reverted
        at getResult (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:191:21)
        at processJsonFunc (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:351:22)
        at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:283:46
        at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
        at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
        at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
        at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
      code: -32000,
      data: undefined
    },
    requestBody: '{"method":"eth_call","params":[{"to":"0xeee0a8f87d358672708598714c84f3e2b741019c","data":"0xe445e7dd"},"latest"],"id":159,"jsonrpc":"2.0"}',
    requestMethod: 'POST',
    url: 'https://alfajores-forno.celo-testnet.org'
  }
}
Failed to derive undefined hook (0x221FA9CBaFcd6c1C3d206571Cf4427703e023FFa): Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data="0x", transaction={"to":"0x221FA9CBaFcd6c1C3d206571Cf4427703e023FFa","data":"0xe445e7dd","accessList":null}, error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":170,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n","error":{"code":-32000},"requestBody":"{\"method\":\"eth_call\",\"params\":[{\"to\":\"0x221fa9cbafcd6c1c3d206571cf4427703e023ffa\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":170,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://alfajores-forno.celo-testnet.org"}, code=CALL_EXCEPTION, version=providers/5.7.2)
    at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
    at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
    at checkError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:108:16)
    at HyperlaneJsonRpcProvider.<anonymous> (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:751:47)
    at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:48:23)
    at Object.throw (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:29:53)
    at rejected (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:21:65)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  reason: 'missing revert data in call exception; Transaction reverted without a reason string',
  code: 'CALL_EXCEPTION',
  data: '0x',
  transaction: {
    to: '0x221FA9CBaFcd6c1C3d206571Cf4427703e023FFa',
    data: '0xe445e7dd',
    accessList: null
  },
  error: Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":170,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_call\",\"params\":[{\"to\":\"0x221fa9cbafcd6c1c3d206571cf4427703e023ffa\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":170,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://alfajores-forno.celo-testnet.org", code=SERVER_ERROR, version=web/5.7.0)
      at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
      at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
      at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:308:32
      at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
      at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
      at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
    reason: 'processing response error',
    code: 'SERVER_ERROR',
    body: '{"jsonrpc":"2.0","id":170,"error":{"code":-32000,"message":"execution reverted"}}\n',
    error: Error: execution reverted
        at getResult (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:191:21)
        at processJsonFunc (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:351:22)
        at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:283:46
        at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
        at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
        at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
        at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
      code: -32000,
      data: undefined
    },
    requestBody: '{"method":"eth_call","params":[{"to":"0x221fa9cbafcd6c1c3d206571cf4427703e023ffa","data":"0xe445e7dd"},"latest"],"id":170,"jsonrpc":"2.0"}',
    requestMethod: 'POST',
    url: 'https://alfajores-forno.celo-testnet.org'
  }
}
EvmCoreReader: readerCall failed for 0xEf9F292fcEBC3848bF4bB92a96a04F9ECBb78E59: Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data="0x", transaction={"to":"0x221FA9CBaFcd6c1C3d206571Cf4427703e023FFa","data":"0xe445e7dd","accessList":null}, error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":170,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n","error":{"code":-32000},"requestBody":"{\"method\":\"eth_call\",\"params\":[{\"to\":\"0x221fa9cbafcd6c1c3d206571cf4427703e023ffa\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":170,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://alfajores-forno.celo-testnet.org"}, code=CALL_EXCEPTION, version=providers/5.7.2)
    at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
    at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
    at checkError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:108:16)
    at HyperlaneJsonRpcProvider.<anonymous> (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:751:47)
    at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:48:23)
    at Object.throw (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:29:53)
    at rejected (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:21:65)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  reason: 'missing revert data in call exception; Transaction reverted without a reason string',
  code: 'CALL_EXCEPTION',
  data: '0x',
  transaction: {
    to: '0x221FA9CBaFcd6c1C3d206571Cf4427703e023FFa',
    data: '0xe445e7dd',
    accessList: null
  },
  error: Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":170,\"error\":{\"code\":-32000,\"message\":\"execution reverted\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_call\",\"params\":[{\"to\":\"0x221fa9cbafcd6c1c3d206571cf4427703e023ffa\",\"data\":\"0xe445e7dd\"},\"latest\"],\"id\":170,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://alfajores-forno.celo-testnet.org", code=SERVER_ERROR, version=web/5.7.0)
      at Logger.makeError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:238:21)
      at Logger.throwError (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/logger/lib/index.js:247:20)
      at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:308:32
      at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
      at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
      at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
    reason: 'processing response error',
    code: 'SERVER_ERROR',
    body: '{"jsonrpc":"2.0","id":170,"error":{"code":-32000,"message":"execution reverted"}}\n',
    error: Error: execution reverted
        at getResult (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:191:21)
        at processJsonFunc (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:351:22)
        at /Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:283:46
        at step (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:33:23)
        at Object.next (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:14:53)
        at fulfilled (/Users/nbayindirli/workplace/Hyperlane/hyperlane-monorepo-official/node_modules/@ethersproject/web/lib/index.js:5:58)
        at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
      code: -32000,
      data: undefined
    },
    requestBody: '{"method":"eth_call","params":[{"to":"0x221fa9cbafcd6c1c3d206571cf4427703e023ffa","data":"0xe445e7dd"},"latest"],"id":170,"jsonrpc":"2.0"}',
    requestMethod: 'POST',
    url: 'https://alfajores-forno.celo-testnet.org'
  }
}
✅ Warp route config written successfully to ./configs/core-config.yaml:

    owner: "0xfaD1C94469700833717Fa8a3017278BC1cA8031C"
    defaultIsm:
      address: "0x4E4b0BAF2ca98ae780c6F5D2fe8804F87D883078"
      type: staticAggregationIsm
      modules:
        - owner: "0xfaD1C94469700833717Fa8a3017278BC1cA8031C"
          address: "0x6e529469D02308250aC253f07e79F5886C05F3c0"
          type: domainRoutingIsm
          domains:
            sepolia:
              address: "0x8fd1b43D6a2392074E5Ba88e2d876d593ceAD8Ce"
              type: staticAggregationIsm
              modules:
                - address: "0x2183183bdc371c67302097DC60952e6E99484720"
                  type: messageIdMultisigIsm
                  validators:
                    - "0x469F0940684D147Defc44F3647146CB90Dd0BC8E"
                    - "0xb22B65F202558ADF86A8BB2847B76AE1036686a5"
                    - "0xd3C75Dcf15056012a4d74C483A0C6ea11d8c2b83"
                  threshold: 2
                - address: "0x631749C86E90Cea9cF3Fb7686eBb6E80EfC9064c"
                  type: merkleRootMultisigIsm
                  validators:
                    - "0x469F0940684D147Defc44F3647146CB90Dd0BC8E"
                    - "0xb22B65F202558ADF86A8BB2847B76AE1036686a5"
                    - "0xd3C75Dcf15056012a4d74C483A0C6ea11d8c2b83"
                  threshold: 2
              threshold: 1
            bsctestnet:
              address: "0x0f0f626cFcea01462ef6CDaF3655d6AF2b175073"
              type: staticAggregationIsm
              modules:
                - address: "0x6b6bEF043905810994a60Cd08F7CdF5BB798f19c"
                  type: messageIdMultisigIsm
                  validators:
                    - "0x1f030345963c54Ff8229720dD3A711c15C554AEB"
                    - "0x242d8a855a8c932deC51f7999ae7D1e48b10C95E"
                    - "0xf620F5e3D25a3Ae848Fec74bccAE5de3edcd8796"
                  threshold: 2
                - address: "0xf75B0F727f1dea2FA491caD2Faf8f3B04B705294"
                  type: merkleRootMultisigIsm
                  validators:
                    - "0x1f030345963c54Ff8229720dD3A711c15C554AEB"
                    - "0x242d8a855a8c932deC51f7999ae7D1e48b10C95E"
                    - "0xf620F5e3D25a3Ae848Fec74bccAE5de3edcd8796"
                  threshold: 2
              threshold: 1
            fuji:
              address: "0x53461D61c69EdEA24ec3A2c62eDf78e9F966a66D"
              type: staticAggregationIsm
              modules:
                - address: "0x3c9eD95aD9a3613a7c036F44fB6745052861f74C"
                  type: messageIdMultisigIsm
                  validators:
                    - "0x43e915573D9f1383cbf482049e4a012290759E7F"
                    - "0x895Ae30Bc83fF1493b9Cf7781b0b813D23659857"
                    - "0xD8154f73D04CC7f7F0c332793692E6E6F6B2402e"
                  threshold: 2
                - address: "0x8d3cf154259E3aF61BA8F62410d9D0E644c6A19f"
                  type: merkleRootMultisigIsm
                  validators:
                    - "0x43e915573D9f1383cbf482049e4a012290759E7F"
                    - "0x895Ae30Bc83fF1493b9Cf7781b0b813D23659857"
                    - "0xD8154f73D04CC7f7F0c332793692E6E6F6B2402e"
                  threshold: 2
              threshold: 1
            scrollsepolia:
              address: "0x45eb8EE25afe6391EA6a42aAdac13319D443359F"
              type: staticAggregationIsm
              modules:
                - address: "0x0caB24DBBDcA1E1bc27dd33481c09d9b306AE4a1"
                  type: messageIdMultisigIsm
                  validators:
                    - "0x7867bea3C9761fE64e6d124b171F91Fd5dD79644"
                    - "0x9a11ed23Ae962974018AB45BC133cAABff7B3271"
                    - "0xbE18Dbd758afb367180260b524E6d4BCd1Cb6d05"
                  threshold: 2
                - address: "0xf8afa14F1cd8600d802C5a82A4406ca83629FC23"
                  type: merkleRootMultisigIsm
                  validators:
                    - "0x7867bea3C9761fE64e6d124b171F91Fd5dD79644"
                    - "0x9a11ed23Ae962974018AB45BC133cAABff7B3271"
                    - "0xbE18Dbd758afb367180260b524E6d4BCd1Cb6d05"
                  threshold: 2
              threshold: 1
        - address: "0xF2bF4407fEf8Fe850F3d24A6224F64d51afd2C84"
          owner: "0xfaD1C94469700833717Fa8a3017278BC1cA8031C"
          type: pausableIsm
          paused: false
      threshold: 2
```